CacheService/docker_context/handlers/CacheHandler.py

```diff
@@ -0,0 +1,154 @@
+import pickle
+from datetime import datetime
+from sanic import Sanic, response
+
+
+def errorResult(err, data):
+    return {
+        'errorMessage': err,
+        'content': data
+    }
+
+
+def goodResult(msg, data):
+    return {
+        'message': msg,
+        'content': data
+    }
+
+
+class CacheHandler:
+
+    def __init__(self, redis, RedisHandler, audit, client, ip, port, LegacyStock):
+        self.LegacyStock = LegacyStock
+        self.redis = redis
+        self.RedisHandler = RedisHandler
+        self.audit = audit
+        self.client = client
+        self.dbmURL = f'http://{ip}:{port}'
+        self.RedisHandler.fillUserCache()
+
+    async def getUserFunds(self, user_id):
+        result = await self.RedisHandler.rExists(user_id)
+        if result:
+            data = await self.RedisHandler.rGet(user_id)
+            return response.json(goodResult(msg="User Data", data=data['account_balance']), status=200)
+        else:
+            result = await self.RedisHandler.updateAccountCache(user_id)
+            if result.status == 200:
+                data = await self.RedisHandler.rGet(user_id)
+                return response.json(goodResult(msg="User Data", data=data['account_balance']), status=200)
```

Comment:
do we want to return the response.json object for these handler calls or the json objects? Just thinking for consistency between the other services where the response type is defined at the api level.